### DISCUSSION TITLE
blog/nested-route/introducing-multi-part-posts-with-nested-routing

### DISCUSSION THREAD

**Discussion by giscus** [ByeongJunAn/ByeongJunAn.github.io#4] [Announcements]
# blog/nested-route/introducing-multi-part-posts-with-nested-routing

The blog template supports posts in nested sub-folders. This can be used to group posts of similar content e.g. a multi-part course. This post is itself an example of a nested route!

https://abj.vercel.app/blog/nested-route/introducing-multi-part-posts-with-nested-routing

**Comment by ByeongJunAn** [comment]
댓글 테스트

**Comment by ByeongJunAn** [comment]
댓글 테스트2
